ts/Dashboard/Component/HTMLComponent.ts

```diff
@@ -196,6 +196,7 @@ class HTMLComponent extends Component<HTMLComponent.HTMLComponentEvents> {
     public render(): this {
         this.emit({ type: 'beforeRender' });
         super.render(); // Fires the render event and calls load
+        this.contentElement.style.overflow = 'auto';
```

Comment:
I guess the scrollbar is ok but since it is not blocking anything we should let @sebastianbochan make a final call.